kornia/augmentation/_2d/geometric/__init__.py

```diff
@@ -8,7 +8,7 @@
 from kornia.augmentation._2d.geometric.perspective import RandomPerspective
 from kornia.augmentation._2d.geometric.resize import LongestMaxSize, Resize, SmallestMaxSize
 from kornia.augmentation._2d.geometric.resized_crop import RandomResizedCrop
-from kornia.augmentation._2d.geometric.rotation import RandomRotation
+from kornia.augmentation._2d.geometric.rotation import RandomRot90, RandomRotation
```

Comment:
```suggestion
from kornia.augmentation._2d.geometric.rotation import RandomRotation90, RandomRotation
```
I'd make naming consistant